backends/backends.c

```diff
@@ -900,43 +1043,77 @@ void *backends_main(void *ptr) {
 
 #if HAVE_MONGOC
         if(do_mongodb) {
+            for(int i = 0; i < MONGODB_THREADS_NUMBER; i++) {
+                netdata_mutex_lock(&mongodb_threads[i].mutex);
+                if(mongodb_threads[i].finished) {
+                    if(unlikely(mongodb_threads[i].error)) {
+                        // oops! we couldn't send (all or some of the) data
+                        error("BACKEND: failed to write data to database backend '%s'. Willing to write %zu bytes, wrote %zu bytes.",
+                              mongodb_uri, mongodb_threads[i].n_bytes, 0UL);
```

Comment:
The `error` is a definition  for the function error_int present inside the file libnetdata/log/log.c , inside this function we have a call for the vfprintf to format the message that will be redirect for our error log, case you see any source of vfprintf you will be that every time that it finds a `%`, it will go to a switch to identify the function that will be called to convert that specific value to a string, considering that the last argument will be awlays the constant `0`, I will give a suggestion that you write the zero direct in the message to avoid this whole process to convert 0 to '0x30' that is its ascii value.